src/store/datasets-files.ts

```diff
@@ -12,23 +59,23 @@ export const useFileStore = defineStore('files', {
   state: (): State => ({
     byDataID: {},
   }),
+
   getters: {
-    getFiles: (state) => {
-      return (dataID: string) => {
-        if (dataID in state.byDataID) {
-          return state.byDataID[dataID];
-        }
-        return null;
-      };
-    },
+    getDatasetFiles: (state) => (dataID: string) =>
+      state.byDataID[dataID] ?? [],
+
+    getFiles: (state) => (dataID: string) =>
```

Comment:
Adding function comments here will help readers differentiate between what `getFiles` and `getDatasetFiles` does. Alternatively, maybe we get rid of `getFiles`, and rewrite usages of that function to use `getDatasetFiles`.

Reply:
Added some basic comments.  After "ingestion," App does not really care about `DatasetFiles` so might be nice to keep the `getFiles` helper around.